Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .gitignore	(date 1496307055000)
+++ .gitignore	(revision )
@@ -5,5 +5,4 @@
 /.idea/libraries
 .DS_Store
 /build
-/captures
+/captures
\ No newline at end of file
-.externalNativeBuild
\ No newline at end of file
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/misc.xml	(date 1496307055000)
+++ .idea/misc.xml	(revision )
@@ -1,0 +1,0 @@
